### ISSUE TITLE
Issue running tests

### ISSUE THREAD

**Issue by AlecRust** [cucumber-ltd/bbc-a11y#45]
After following your readme instructions we're having issues running these accessibility tests.

The tests seem to run after setting up the `Gemfile`, installing dependencies and creating the `.a11y.rb` config file, but each test isn't able to complete the "I visit the page" step, resulting in a test run output like:

```
Correctly use headings
----------------------

  - Check headings... ?

      When I visit the page
      Then there must be exactly one h1 element
      And each heading must be followed by content or a heading of one level deeper (h2-h6)

    Undefined step: "I visit the page"


Correctly use `title` attributes
--------------------------------

  - Check the elements with title attributes... ?

      When I visit the page
      Then there must be no elements with a title attribute whose content is repeated within the element
      And any form fields with associated labels do not have a title attribute

    Undefined step: "I visit the page"


Correctly use `tabindex` attributes
-----------------------------------

  - Check all tabindex values... ?

      When I visit the page
      Then there should be no elements with a tabindex attribte of 0 or greater

    Undefined step: "I visit the page"
```

Can you shed any light on this? A couple members of our team have tried this with the same results.

**Comment by mattwynne** [comment]
Hi @AlecRust - thanks for the report.

I've fixed this and released v0.0.11 with the fix. Can you give it a try? Use `bundle update bbc-a11y` to get the latest.

**Comment by AlecRust** [comment]
Hi @mattwynne thanks for that! I'm now able to run the tests which work well when run against URLs like `http://www.bbc.co.uk/bbcthree` or `http://www.google.co.uk`.

However when I try some other BBC pages such as `http://www.bbc.co.uk` or the Int environment of our project, PhantomJS has some JS errors which result in test result like the below.

I wonder if it's worth defining as part of this project which driver is required to run these tests?

```
Specify content language
------------------------

  - Check main body element lang attribute... 2015-09-04 13:43:50.815 phantomjs[35947:612605] *** WARNING: Method userSpaceScaleFactor in class NSView is deprecated on 10.7 and later. It should not be used in new applications. Use convertRectToBacking: instead.
✗

      When I visit the page
      Then the <html> element must have a `lang` attribute
      And the main natural language of the page must match that attribute

    One or more errors were raised in the Javascript code on the page. If you don't care about these errors, you can ignore them by setting js_errors: false in your Poltergeist configuration (see documentation for details).

    TypeError: 'undefined' is not a function (evaluating 's.createElement.bind(null,e)')
    TypeError: 'undefined' is not a function (evaluating 's.createElement.bind(null,e)')
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:11
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:10 in o
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:14 in o
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:10
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:9
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:14
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:6
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:6
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in e
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:14


  - Check for other elements with lang attributes... ✗

      When I visit the page
      Then all elements with `lang` attribute must have content in that natural language

    One or more errors were raised in the Javascript code on the page. If you don't care about these errors, you can ignore them by setting js_errors: false in your Poltergeist configuration (see documentation for details).

    TypeError: 'undefined' is not a function (evaluating 's.createElement.bind(null,e)')
    TypeError: 'undefined' is not a function (evaluating 's.createElement.bind(null,e)')
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:11
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:10 in o
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:14 in o
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:10
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:9
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:14
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:6
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:6
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in e
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:14


  - Check for areas of the page expressed in other languages, but missing a `lang` attribute... ✗

      When I visit the page
      Then any parts expressed in a natural language different to the main language of the page must have a matching `lang` attribute

    One or more errors were raised in the Javascript code on the page. If you don't care about these errors, you can ignore them by setting js_errors: false in your Poltergeist configuration (see documentation for details).

    TypeError: 'undefined' is not a function (evaluating 's.createElement.bind(null,e)')
    TypeError: 'undefined' is not a function (evaluating 's.createElement.bind(null,e)')
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:11
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:10 in o
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:14 in o
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:10
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:9
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:14
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:6
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:6
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in r
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:3 in e
        at http://a.files.bbci.co.uk/s/homepage-v5/1594/javascripts/app.js:14
```
